app/resources/add_note.html.template

```diff
@@ -8,281 +8,76 @@
 {# See the License for the specific language governing permissions and      #}
 {# limitations under the License.                                           #}
 
-{# Template fragment for the "Add a note" form (on view and create pages).  #}
+{# Template for the add note to record page (see add_note.py).              #}
 
+{% extends "app-base.html.template" %}
 {% load i18n %}
 
-{% if not markdup %}
-  <input type="hidden" name="add_note" value="yes">
-  <input type="hidden" name="id" value="{{params.id}}">
-{% endif %}
+{% block title %}
+  {{person.full_name}} - {{block.super}}
+{% endblock %}
 
-<div class="fields-table note" role="complementary">
-  <h2>
-    {% if not markdup %}
-      {% trans "Tell us the status of this person" %}
-    {% else %}
-      {% trans "Mark records as duplicate" %}
-    {% endif %}
-  </h2>
-  {% if not markdup %}
-    <div class="status section">
-      <h3>
-        {% trans "Status of this person" %}
-      </h3>
-      <div class="field">
-        <select id="status" name="status">
-          {% for status in env.status_options %}
-            <option value="{{status.value}}"
-                {% ifequal params.status status.value %}
-                  selected='selected'
-                {% endifequal %}>
-              {{status.text}}
-            </option>
-          {% endfor %}
-        </select>
-      </div>
-      <div class="end-multi-columns"></div>
-    </div>
+{% block head %}
+  {{block.super}}
+  {% if env.enable_javascript %}
+    <script type="text/javascript">
+      function set_display(id_or_elem, hide) {
+        var hide_text = hide ? 'none' : '';
+        if (typeof(id_or_elem) == 'string') {
+          document.getElementById(id_or_elem).style.display = hide_text;
+        } else {
+          id_or_elem.style.display = hide_text;
+        }
+      }
+
+      function hide_unhide_note_contents(note_contents_id) {
+        var note = document.getElementById(note_contents_id + '-contents');
+        var hidden = note.style.display == 'none';
+        set_display(note, !hidden);
+        set_display('reveal-note', hidden);
```

Comment:
This is the current behavior in the live app, though I do agree it's confusing. I think it would be good to discuss if we actually want to change this behavior (if a record has 50 notes, we probably don't want someone to have to enter a captcha for each note) and work on it in a separate pull request.